task4/Dogs/Dogs.txt

```diff
@@ -0,0 +1,6 @@
+list/all
```

Comment:
Please never leave comments in Russian
Moreover never leave them not in Unicode))
```
image/random
�������� ���� �� ������� �� ��������� �����������
```